### PR TITLE
feat ✨: Allow to create requests projects.

### PR DESCRIPTION
**Pull request by joaquingx**
## PR Changes:
- [x] Allow to create requests projects.
- [x] Update python version.

### CODE REVIEW

File: estela_cli/init.py [hunk 1]
```diff
@@ -21,10 +22,10 @@
 )
 
 
-SHORT_HELP = "Initialize estela project for existing scrapy project"
+SHORT_HELP = "Initialize estela project for existing crawling project"
```
Comment (breno) [inline estela_cli/init.py:25]: @joaquingx I think web scraping is probably a more standard term here (regardless of technicalities). cc: @mgonnav

Comment (joaquingx) [reply estela_cli/init.py]: I agree, done.

File: estela_cli/init.py [hunk 2]
```diff
@@ -85,22 +86,32 @@ def gen_dockerfile(requirements_path):
...
+    }
     if not os.path.exists(ESTELA_DIR):
         os.makedirs(ESTELA_DIR)
-
     estela_client = login()
-    gen_estela_yaml(estela_client, pid)
-    gen_dockerfile(requirements)
+    gen_estela_yaml(estela_client, platform_map[platform], pid)
+    gen_dockerfile(requirements, platform_map[platform])
+    click.echo(f"{pid} project is activated as a {platform.capitalize()} project.")
```
Comment (breno) [inline estela_cli/init.py:117]: @joaquingx initialized instead of activated

File: estela_cli/create/project.py [hunk 3]
```diff
@@ -19,8 +19,9 @@ def estela_command(name):
         response = estela_client.create_project(name)
         click.echo("project/{} created.".format(name))
         click.echo(
-            "Hint: Run 'estela init {}' to activate this project".format(
-                response["pid"]
+            "Hint: Use 'estela init {}' to activate this project as a Scrapy project.\nHint: Use 'estela init {} -p requests' to activate it as a Requests project.".format(
```
Comment (breno) [inline estela_cli/create/project.py:22]: @joaquingx initialize instead of activate (there's two instances)

Comment (joaquingx) [reply estela_cli/create/project.py]: Done!

File: estela_cli/init.py (hunk 2 above)
Comment (joaquingx) [reply estela_cli/init.py]: Done

File: estela_cli/init.py [hunk 4]
```diff
@@ -85,22 +86,32 @@ def gen_dockerfile(requirements_path):
...
+    }
     if not os.path.exists(ESTELA_DIR):
         os.makedirs(ESTELA_DIR)
-
     estela_client = login()
-    gen_estela_yaml(estela_client, pid)
-    gen_dockerfile(requirements)
+    gen_estela_yaml(estela_client, platform_map[platform], pid)
+    gen_dockerfile(requirements, platform_map[platform])
+    click.echo(f"{pid} project is initialized as a {platform.capitalize()} project.")
```
Comment (mgonnav) [inline estela_cli/init.py:117]: If I use `estela init <PID> -p whatever` the CLI breaks. Please specify allowed options for the `-p` option such as in https://github.com/bitmakerla/estela-cli/blob/09db075d5321e5186a7bb020d899fb8d05a605f0/estela_cli/data/job.py#L27

**Review by mgonnav — CHANGES_REQUESTED** [review]
I have additionally changed the default Python version from 3.8 to 3.9 and updated the code style in the `init` hint message. Please handle the options for `--platform` and I think we can merge this.

**Review by mgonnav — APPROVED** [review]
Looks good!

File: estela_cli/init.py [hunk 5]
```diff
@@ -85,22 +86,34 @@ def gen_dockerfile(requirements_path):
...
+        "scrapy": DOCKER_DEFAULT_ENTRYPOINT,
+        "requests": DOCKER_REQUESTS_ENTRYPOINT,
+    }
     if not os.path.exists(ESTELA_DIR):
         os.makedirs(ESTELA_DIR)
-
     estela_client = login()
-    gen_estela_yaml(estela_client, pid)
-    gen_dockerfile(requirements)
+    estela_client.update_project(pid, framework=platform.upper(), action="update")
```
Comment (raymond1242) [inline estela_cli/init.py:116]: Could be useful verify this response for check everything it's OK

Comment (joaquingx) [reply estela_cli/init.py]: Done!

**Comment by mgonnav** [comment]
@joaquingx It might be a good idea to also be able to specify the project type in the `estela create project` command. That can be added later.

**Pull request merged**